java-manta-it/src/test/java/com/joyent/manta/client/MantaClientSigningIT.java

```diff
@@ -65,6 +74,51 @@ public void afterClass() throws IOException {
         IntegrationTestConfigContext.cleanupTestDirectory(mantaClient, testPathPrefix);
     }
 
+
+    // When MantaClient TLS security is disabled (such as when testing against a
+    // lab instance without a self signed cert), we will also need to refrain
+    // from checking the certs on the resulting signed urls.
+    private HttpURLConnection openHttpURLConnection(URI uri, MantaClient client) throws IOException {
+        boolean tlsInsecure = client.getContext().tlsInsecure();
+        HttpURLConnection connection = (HttpsURLConnection)uri.toURL().openConnection();
+        if (tlsInsecure && connection instanceof HttpsURLConnection) {
```

Comment:
Doubt: Could we replace `instanceof` condition with a `null` check ?

Reply:
The `instanceof` is for checking to make sure it is an `HttpsURLConnection` (subclass of `HttpURLConnection`).  I think almost all manta standups -- even lab ones -- use https, but if someone had an http only dev setup I didn't want to break that.